code/game/objects/effects/temporary_visuals/projectiles/projectile_effects.dm

```diff
@@ -36,26 +36,21 @@
 		apply_vars(angle_override, p_x, p_y, color_override, scaling)
 	return ..()
 
-/obj/effect/projectile/proc/apply_vars(angle_override, p_x = 0, p_y = 0, color_override, scaling = 1, atom/new_loc, increment = 0)
-	var/mutable_appearance/look = new(src)
-	SET_PLANE_EXPLICIT(look, plane, new_loc || src)
-	look.pixel_x = p_x
-	look.pixel_y = p_y
+/obj/effect/projectile/proc/apply_vars(angle_override, p_x = 0, p_y = 0, color_override, scaling = 1, new_plane = null, increment = 0)
+	pixel_x = p_x
+	pixel_y = p_y
```

Comment:
why do we have the new_plane var if it's gonna be discarded?
unrelated, why nuke the MA here?

Reply:
no reason to use MAs, new_plane is a leftover